.github/workflows/cypress-next-11.yml

```
@@ -0,0 +1,73 @@
+name: Run e2e (Next 11 demo)
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled, synchronize]
+  push:
+    branches:
+      - main
+    paths:
+      - 'demos/next-11/**/*.{js,jsx,ts,tsx}'
+      - 'cypress/integration/next-11/**/*.{ts,js}'
+      - 'src/**/*.{ts,js}'
+jobs:
+  cypress:
+    name: Cypress
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        containers: [1, 2, 3, 4]
```

If there's not many tests that are included in this suite, it's likely not worth having so many containers dedicated to this. However, I'd leave it as a `matrix` so that as we add more tests it's really easy to add containers to keep the time to run the tests low.

```suggestion
        containers: [1]
```